qiita_pet/webserver.py

```diff
@@ -57,6 +57,16 @@
     ReloadPluginAPItestHandler)
 from qiita_pet import uimodules
 from qiita_db.util import get_mountpoint
+from qiita_pet.handlers.rest import (
+    StudyHandler as WeDontKnowWhatToCallThis,
```

Comment:
🍻 I think that is an awesome solution! Never though about that and it is superclean!